src/index.ts

```diff
@@ -302,44 +380,54 @@ interface NamespacedDeclaration {
   parentNamespace: NamespacedContext
 }
 
-function getInterface (interfacePath: string, interfaceName: string): InterfaceDeclaration | undefined {
+function parseDefinitionFiles (interfacePath: string): SourceFile | undefined {
   const interfaceFile = ast.addExistingSourceFile(interfacePath)
   if (!interfaceFile) return
 
   trackUserAddedDefinitionFile(interfaceFile)
   for (const file of ast.resolveSourceFileDependencies()) {
     trackUserAddedDefinitionFile(file)
   }
-
-  return getNamespacedInterface.call(this, interfaceFile, interfaceName)
+  return interfaceFile
 }
 
-function getNamespacedInterface (
-  rootNamespace: NamespacedContext,
-  interfaceName: string
-): InterfaceDeclaration | undefined {
-  const isNamespacedInterface = interfaceName.match(/(?:[a-zA-Z0-9_]\.)*[a-zA-Z0-9_]\./i)
+function extractNamespace (
+    rootNamespace: NamespacedContext,
```

Comment:
why tabulating with 4 spaces this function? It looks like the rest of the file is using 2